processor/deltatocumulativeprocessor/internal/data/data.go

```diff
@@ -80,10 +80,11 @@ func (dp Histogram) CopyTo(dst Histogram) {
 
 type ExpHistogram struct {
 	expo.DataPoint
+	MaxSize int
```

Comment:
This doubles the size of each `ExpHistogram` value from 8 to 16.

Given there is likely no need to configure this at runtime anyways, can we just have a constant on the expo package?

```
expo.Limit(dp, in, expo.DefaultLimit)